Sources/Logging/Strings/AttributionStrings.swift

```diff
@@ -44,6 +44,7 @@ enum AttributionStrings {
     case adservices_token_post_succeeded
     case adservices_token_unavailable_in_simulator
     case latest_attribution_sent_user_defaults_invalid(networkKey: String)
+    case copying_attributes_from_to_user(oldAppUserID: String, newAppUserID: String)
```

Comment:
Hmm when I try that, I get `Enum case cannot have keyword arguments`. Looking around, I don't think it's supported on enums? Going to rename it to `copying_attributes(oldAppUserID: String, newAppUserID: String)` though, since that's probably clearer.

Reply:
Oh yes sorry, that works